src/Utils.tsx

```diff
@@ -0,0 +1,25 @@
+import { Fraction } from 'api';
+
+export function gradeToString(grade: number | Fraction | undefined) {
```

Comment:
Import the grade type and specify:

```suggestion
export function gradeToString(grade?: Grade) {
```

Reply:
Are we making grade optional so that null can be returned without being passed in?

Reply:
Fixed. Should not be optional.